routes/index.js

```diff
@@ -1,7 +1,11 @@
 const { apiErrorHandler } = require('../middleware/error-handler');
 const testController = require('../controllers/test-controller');
+const express = require('express');
+const router = express.Router();
```

Comment:
make all "require"(except dynamic importing) on the top

Reply:
done